src/main/kotlin/net/dungeonhub/application/enums/HelpTopic.kt

```diff
@@ -156,14 +156,12 @@ enum class HelpTopic(
                         "- Join any lobby using `/lobby`\n" +
                         "- Right click with the \"My Profile\" item (second hotbar slot) in your hand.\n" +
                         "- Select \"Social Media\", to the right of your player head, then select \"Discord\".\n" +
-                        "- Paste your Discord username (" + user.username + ") exactly like that into the chat.\n" +
+                        "- Paste your Discord username (**" + user.username + "**) exactly like that into the chat.\n" +
                         "- If you get shown a book, select \"I understand\".\n" +
                         "## 2. On Discord\n" +
                         "- Use the `/link` command, this should complete the process.\n\n" +
-                        "Please keep in mind that the API can take up to 15 minutes to update the changes made " +
-                        "on Hypixel. If you have an issue with verification and only recently changed your " +
-                        "settings, wait a few minutes and try again!\n" +
-                        "\n" +
+                        "**It may take up to 15 minutes for your stats to update**, please wait and try again if verification doesn't work right away.\n\n" +
```

Comment:
Keep a blank line to be able to easier see in the editor when theres a blank line.
```suggestion
                        "**It may take up to 15 minutes for your stats to update**, please wait and try again if verification doesn't work right away.\n" +
                        "\n" +
```